demo/startRedis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Starting redis-server as a UNIX daemon process and
+# putting the process to sleep state for 1 second
+redis-server --daemonize yes && sleep 1
+
+# Adding default hashlist to the redis-server
+redis-cli < /usr/local/hashlist.redis
+
+# Saving the added entries
+redis-cli save
+
+# closing the cli prompt
```

Comment:
Can be rephrased as 'Stopping the 'redis-server' instance' ?

Reply:
Updated the comment